File: Python/cuenta.py
class Cuenta:
    id       = int
    name     = str
    document = str
    email    = str
    password = str
    number   = str

    def __init__(self, id, name, document, email, password, number):
        self.id = id
        self.name = name
        self.document = document
        self.email = email
        self.password = password
        self.number = number

File: PHP/cuenta.php
<?php
class cuenta {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;
    public $number;

    public function __construct($id, $name, $document, $email, $password, $number) {
    $this->id = $id;
    $this->name = $name;
    $this->document = $document;
    $this->email = $email;
    $this->password = $password;
    $this->number = $number;
    }
}
?>